Exiled.API/Extensions/EffectTypeExtension.cs

```diff
@@ -5,6 +5,8 @@
 // </copyright>
 // -----------------------------------------------------------------------
 
+using InventorySystem.Items.MarshmallowMan;
```

Comment:
????

Reply:
marshmallow effect there. idk why

Reply:
i mean it's not in the right place of using lol

Reply:
yeah. I'll edit it